spec.md

```diff
@@ -129,6 +129,19 @@ variant type.
 Every CloudEvent conforming to this specification MUST include one or more
 of the following context attributes.
 
+It could also include additional attributes in the "context attributes" that
+might be used in ancillary actions related to the processing of the event.
+For example, in many IoT and enterprise use cases, an event could be used in
+a serverless application that performs actions across multiple types of events.
+To support such use cases, the event producer will need to add additional
+identity attributes to the "context attributes" which the event consumers can
+use to correlate this event with the other events. If such identity attributes
+happen to be part of the event "data", it is still suggested that the event
+producer add the identity attributes to the "context attributes" so that event
```

Comment:
`also adds` may be clearer - the attribute shouldn't be removed from "data".

I'm wondering if a more generic version of this applies to all extension attributes - it is completely fine if an attribute is "duplicated" in data and context.

Reply:
Agree it is completely fine if an attribute is duplicated in data and context and this applies to all attributes including extension attributes. Will use "also add" to be clearer.